tutorials/13_Submit_to_IIASA_database.md

```diff
@@ -23,16 +23,14 @@ You can generate the file to be uploaded by either calling [`piamInterfaces::gen
 - `iiasatemplate`: optional path to the xlsx or yaml file obtained in the project with the variables and units that are accepted in the database
 - `addToScen`: optional string added in front of all scenario names
 - `removeFromScen`: optional regular expression of parts to be deleted from the scenario names, such as "C_|_bIT|_bit|_bIt"
-- you have to specify at least one of:
-  - `mappingFile`: filename of the csv file generated by [`piamInterfaces::generateMappingfile`](https://github.com/pik-piam/piamInterfaces/blob/master/R/generateMappingfile.R) that is used. If `mapping` is also supplied, this file is overwritten
-  - `mapping`: vector of templates from [this directory](https://github.com/pik-piam/piamInterfaces/tree/master/inst/templates) (such as `c("AR6", "AR6_NGFS")` or `c("NAVIGATE", "SHAPE")`) that then generates the template (and writes it to `mappingFile` if defined)
+- `mapping`: vector of templates from [this directory](https://github.com/pik-piam/piamInterfaces/tree/master/inst/templates) (such as `c("AR6", "AR6_NGFS")` or `c("NAVIGATE", "SHAPE")`) or a local file with identical structure.
```

Comment:
in theory, could also be a vector of file paths that will be concatenated as well, right?